nix/flakebox.nix

```diff
@@ -357,6 +357,12 @@ rec {
     buildPhaseCargoCommand = "patchShebangs ./scripts ; ./scripts/tests/latency-test.sh";
   };
 
+  guardianBackupTest = craneLibTests.mkCargoDerivation {
```

Comment:
For devimint tests should we continue adding these? I believe we're only using `ciTestAllBase` which will handle calling each individual test.

cc: @dpc